### PR TITLE
Set up project to be uploadable to Pypi

### PR DESCRIPTION
**Pull request by Guptacos**
- Update gitignore for setuptools builds
    - ignore build, dist, and *.egg-info
- Add instructions to DEV_README for how to build and upload a package.
- Update requirements.txt with modules needed to build the package
  (twine and dependencies)
- Create setup.py to describe how to build the package

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        'Development Status :: 1 - Planning',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: MIT License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+    ],
+    # Use ~= to signify that we aren't commiting to Python 4 support yet
```
Comment (spenceryu) [inline setup.py]: I thought python 4 isn't released yet, why is ~= on 3.6?

File: requirements.txt [hunk 2]
```diff
@@ -1,10 +1,28 @@
-wheel==0.34.2
 astroid==2.4.1
```
Comment (spenceryu) [inline requirements.txt:7]: are most of these requires meant for dependencies/build? not too sure on what they actually do.

**Review by spenceryu — DISMISSED** [review]

**Review by rmannava — DISMISSED** [review]

File: requirements.txt (hunk 2 above)
Comment (Guptacos) [reply requirements.txt:7]: These are requirements for building the actual spotifython package and for building documentation. It'll only be used during dev

File: setup.py (hunk 1 above)
Comment (Guptacos) [reply setup.py]: See [here](https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires)

Do you think we should just change it to `>=`?

**Pull request merged**